### PR TITLE
spider: grammatical error fix in help content

### PR DESCRIPTION
**Pull request by njmulsqb**
Grammatical Error Fix

~ 💚 from Tecvity

### CODE REVIEW

**Comment by njmulsqb** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by kingthorin — APPROVED** [review]
Thanks

**Comment by thc202** [comment]
Changelog should be updated.

**Comment by thc202** [comment]
Bump.

**Comment by njmulsqb** [comment]
> Changelog should be updated.

Can you mention the path of this file?

**Comment by kingthorin** [comment]
It's in the root of the add-on folder: CHANGELOG.md

**Comment by thc202** [comment]
https://github.com/zaproxy/zap-extensions/blob/07e2410498d0e7e3107caa70930aa407a2ad4187/addOns/spider/CHANGELOG.md

File: addOns/spider/CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 - Ensure issues in one parser don't break the whole parsing process.
 - Fix exception that happened with absolute dotted URLs in inlined content.
+- Fix grammatical error in help content.
```
Comment (thc202) [inline addOns/spider/CHANGELOG.md:19]: This should be added to the Unreleased section not a version already released.

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**